src/govuk/components/skip-link/skip-link.js

```diff
@@ -83,6 +94,11 @@ SkipLink.prototype.removeFocusProperties = function ($linkedElement) {
  * @returns {string} Fragment from URL, without the hash
  */
 SkipLink.prototype.getFragmentFromUrl = function (url) {
+  // Bail if the url is not a string (for example if it's undefined)
```

Comment:
Just noticed that there's a method [`HTMLAnchorElement.hash`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLAnchorElement/hash) which we might be able to use here to simplify things a little.

Reply:
I've had a go at using it, have a look. I removed the check for the type of `string` as the native check for the hash on an anchor link should handle that.